Resources/Private/JavaScript/Host/Containers/LeftSideBar/index.js

```diff
@@ -10,17 +10,20 @@ import style from './style.css';
 const {$get} = immutableOperations;
 
 @connect(state => ({
-    isHidden: $get(state, 'ui.leftSideBar.isHidden')
+    isHidden: $get(state, 'ui.leftSideBar.isHidden'),
+    isFullScreen: $get(state, 'ui.fullScreen.isFullScreen')
 }))
 export default class LeftSideBar extends Component {
     static propTypes = {
-        isHidden: PropTypes.bool.isRequired
+        isHidden: PropTypes.bool.isRequired,
+        isFullScreen: PropTypes.bool.isRequired
     };
 
     render() {
         const classNames = mergeClassNames({
             [style.leftSideBar]: true,
-            [style['leftSideBar--isHidden']]: this.props.isHidden
+            [style['leftSideBar--isHidden']]: this.props.isHidden,
```

Comment:
That's exactly how I had that originally implemented: in the connect. But it didn't work with the RightSideBar, as when it is hidden it still has that chevron toggle shown. Thanks for the hint about repaint, didn't think about it.
So do I just add some extra CSS to hide the toggle button for the rightSideBar or what?